catalog/nsili/catalog-nsili-source/src/main/java/org/codice/alliance/nsili/source/NsiliSource.java

```diff
@@ -662,7 +655,7 @@ public void refresh(Map<String, Object> configuration) {
         LOGGER.debug("Entering Refresh : {}", getId());
 
         if (MapUtils.isEmpty(configuration)) {
-            LOGGER.error("{} {} : Received null or empty configuration during refresh.",
+            LOGGER.debug("{} {} : Received null or empty configuration during refresh.",
                     this.getClass()
```

Comment:
Seems like overkill to log the classname when the log message itself should include the classname/line numer.